From 0f828ef8886c92c8f5dab950bad33a3aa0a112cf Mon Sep 17 00:00:00 2001
From: Paul Durrant <paul.durrant@citrix.com>
Date: Fri, 15 Feb 2019 09:57:01 +0000
Subject: [PATCH] tools/libxendevicemodel: add
 xendevicemodel_modified_memory_bulk to map

Commit e3b93b3c595 "dmop: add xendevicemodel_modified_memory_bulk()" added
the implementation to the library almost 2 years ago, but the function
was not included in the map file, essentially making it useless. This
patch rectifies the situation.

Signed-off-by: Paul Durrant <paul.durrant@citrix.com>
---
 tools/libs/devicemodel/Makefile              | 2 +-
 tools/libs/devicemodel/libxendevicemodel.map | 5 +++++
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/tools/libs/devicemodel/Makefile b/tools/libs/devicemodel/Makefile
index 5b2df7a18e..73cff6dbc4 100644
--- a/tools/libs/devicemodel/Makefile
+++ b/tools/libs/devicemodel/Makefile
@@ -2,7 +2,7 @@ XEN_ROOT = $(CURDIR)/../../..
 include $(XEN_ROOT)/tools/Rules.mk
 
 MAJOR    = 1
-MINOR    = 2
+MINOR    = 3
 SHLIB_LDFLAGS += -Wl,--version-script=libxendevicemodel.map
 
 CFLAGS   += -Werror -Wmissing-prototypes
diff --git a/tools/libs/devicemodel/libxendevicemodel.map b/tools/libs/devicemodel/libxendevicemodel.map
index 04797b239d..561c62deb4 100644
--- a/tools/libs/devicemodel/libxendevicemodel.map
+++ b/tools/libs/devicemodel/libxendevicemodel.map
@@ -33,3 +33,8 @@ VERS_1.2 {
 		xendevicemodel_relocate_memory;
 		xendevicemodel_pin_memory_cacheattr;
 } VERS_1.1;
+
+VERS_1.3 {
+	global:
+		xendevicemodel_modified_memory_bulk;
+} VERS_1.2;
-- 
2.20.1.2.gb21ebb671

